content/docs/SUSHI/tips/_index.md

```diff
@@ -271,3 +271,34 @@ This IG provides [MyPatient] for patient information.
 
 {% include fsh-link-references.md %}
 ```
+
+## Choosing a SNOMED CT edition
+
+To specify a certain edition of SNOMED CT - for example to ensure terminology validation works correctly - do the following:
+
+1. Add a new `parameters` section to your `sushi-config.yaml`:
+```yml
+parameters:
+  path-expansion-params: Parameters-expansion.json
+```
+2. Create an `input/fsh/expansion.fsh` file. If you'd like to use the US edition of SNOMED, fill this in:
```

Comment:
I think it might be helpful to provide just a little more detail here. Maybe something like:
```md
2. Create an `input/fsh/expansion.fsh` file and define an instance of `Parameters` with the id `expansion`. Within the Parameters instance, specify a `system-version` parameter. For example, if you'd like to use the US edition of SNOMED, add this definition:
```